ontology_processing/graph_creation/make_graph_class.py

```diff
@@ -0,0 +1,652 @@
+import pickle
+import argparse
+import networkx as nx
+import pandas as pd
+import validators
+
+import owlready2
+from owlready2 import sync_reasoner
+from collections import OrderedDict
+
+try:
+    from graph_creation.ontology_processing_utils import (
+        give_alias,
+        save_test_ontology_to_json,
+        save_graph_to_pickle,
+        get_valid_test_ont,
+        get_non_test_ont,
+        remove_non_test_nodes,
+        get_test_ontology,
+        get_source_types,
+        solution_sources,
+        listify,
+        union_subgraph,
+        custom_bfs,
+    )
+except ImportError:
+    from ontology_processing.graph_creation.ontology_processing_utils import (
+        give_alias,
+        save_test_ontology_to_json,
+        save_graph_to_pickle,
+        get_valid_test_ont,
+        get_non_test_ont,
+        remove_non_test_nodes,
+        get_test_ontology,
+        get_source_types,
+        solution_sources,
+        listify,
+        union_subgraph,
+        custom_bfs,
+    )
+
+# Set a lower JVM memory limit
+owlready2.reasoning.JAVA_MEMORY = 500
+
+class MakeGraph:
+    def __init__(self, onto_path, edge_path, output_folder_path="."):
+        self.onto_path = onto_path
+        self.edge_path = edge_path
+        self.output_folder_path = output_folder_path
+        self.onto = None
+        self.object_properties = None
+        self.annot_properties = None
+        self.data_properties = None
+        self.all_myths = None
```

Comment:
If going to have all myths stored, could also have stored all solutions and all effects. And could have all mitigation and all adaptation? Or could remove the all myths and put the all myths similar how solutions and effects are currently stored.

Reply:
Okay I removed myths and put it similar to the others. The less class variables the better and myths is only used once and then put into a "getter" function so it makes no sense to have this here. Thank you